content/zh/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Kubernetes API 健康端点
+reviewers:
+- logicalhan
```

Comment:
Please remove `reviewers` in localized version. They rarely speak Chinese.

Reply:
copy that